### PR TITLE
dropped transactions

### PR DESCRIPTION
**Pull request by georgeburduhos**

### CODE REVIEW

File: src/redux/wallets/reducer.ts [hunk 1]
```diff
@@ -154,6 +155,23 @@ export default (state: IWalletsState = intialState, action: IAction) => {
                 }
             };
 
+        case UPDATE_TRANSACTION_STATUS:
```
Comment (maricalucian) [inline src/redux/wallets/reducer.ts]: You could use `TRANSACTION_UPSERT` for this. Basically it does the same thing, but alsa adds as a new transaction is the transaction does not exists. Or do you think unwanted errorenous insert could be an issue?

Comment (georgeburduhos) [reply src/redux/wallets/reducer.ts]: I wanted to keep separate the transaction updates that come from push notifications and the ones that are based on app calls. This case will update only the status for a specific transaction. @krisboit ?

File: src/core/blockchain/cosmos/config.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ export const config: IBlockchainConfig = {
     coin: 'ATOM',
     defaultUnit: 'UATOM',
     iconComponent: CosmosIcon,
+    nrBlocksPassed: 10,
```
Comment (krisboit) [inline src/core/blockchain/cosmos/config.ts]: we need a more intuitive name: something like `droppedTxBlocksThreshold`

File: src/core/blockchain/ethereum/config.ts [hunk 3]
```diff
@@ -30,6 +30,7 @@ export const config: IBlockchainConfig = {
     coin: 'ETH',
     defaultUnit: 'WEI',
     iconComponent: EthIcon,
+    nrBlocksPassed: 10,
```
Comment (krisboit) [inline src/core/blockchain/ethereum/config.ts]: we need to do some research here, i remember on ethereum we can consider a transaction dropped after 50 blocks

File: src/redux/wallets/reducer.ts (hunk 1 above)
Comment (krisboit) [reply src/redux/wallets/reducer.ts]: if the actions are doing the same thing, let's reuse them.

File: src/core/blockchain/ethereum/config.ts [hunk 4]
```diff
@@ -30,6 +30,7 @@ export const config: IBlockchainConfig = {
     coin: 'ETH',
     defaultUnit: 'WEI',
     iconComponent: EthIcon,
+    droppedTxBlocksThreshold: 10,
```
Comment (krisboit) [inline src/core/blockchain/ethereum/config.ts]: for ethereum let's use 50

**Pull request merged**